target/scripts/check-for-changes.sh

```diff
@@ -86,6 +88,16 @@ function _get_changed_files
   grep -Fxvf "${CHKSUM_CURRENT}" "${CHKSUM_NEW}" | sed -r 's/^\S+[[:space:]]+//'
 }
 
+function _reload_amavis
+{
+  if [[ ${CHANGED} =~ ${DMS_DIR}/postfix-accounts.cf ]] || [[ ${CHANGED} =~ ${DMS_DIR}/postfix-virtual.cf ]]
+  then
+    # /etc/postfix/vhost was updated, amavis must refresh it's config by
+    # reading this file again in case of new domains, otherwise they will be ignored.
+    amavisd-new reload
```

Comment:
Presumably `supervisorctl restart amavis` should be used.

I've [not received any feedback about better understanding why `postfix reload` was an issue in the past](https://github.com/docker-mailserver/docker-mailserver/pull/2624#issuecomment-1148043165) (_may have been due to the present `postfix-wrapper.sh` script?_).

Reply:
Truth be told, I don‘t know 🙈 @casperklein do you have an idea?:)

Reply:
The usage of `amavisd-new reload` should be fine. However, if for some reason a complete service restart is needed, `supervisorctl restart amavis` should be used.